src/hooks/useTxDetails/useTxDetails.ts

```diff
@@ -97,19 +104,33 @@ export const getTransfers = (tx: Tx, assets: AssetsByIdPartial): Transfer[] => {
         ...transfer,
         asset: {
           ...asset,
-          symbol: asset.symbol || transfer.token?.symbol || '',
-          name: asset.name || transfer.token?.name || '',
+          symbol: asset.symbol || transfer.token?.symbol || 'N/A',
+          name: asset.name || transfer.token?.name || 'Unknown',
         },
       })
     } else {
-      prev.push({ ...transfer, asset: defaultAsset })
+      const minimalAsset: MinimalAsset = {
```

Comment:
create minimal asset and upsert for missing assets from transfers (example, we aren't indexing nfts on base, so there is no asset upsert based on portfolio data, but this can fill in the gaps) if not generated or upserted anywhere else.